src/Build/Logging/BinaryLogger/BinaryLogger.cs

```diff
@@ -98,6 +151,24 @@ private void Write(BuildEventArgs e)
                 {
                     eventArgsWriter.Write(e);
                 }
+
+                IncludeSourceFilesFrom(e);
+            }
+        }
+
+        private void IncludeSourceFilesFrom(BuildEventArgs e)
```

Comment:
Renamed to `CollectImports` because `CollectProjectImports` is already a property of type `ProjectimportsCollectionMode`